etc/s3-default.cfg

```diff
@@ -8,7 +8,7 @@ eventlet_debug = true
 sds_default_account = AUTH_demo
 
 [pipeline:main]
-pipeline = catch_errors cache s3api tempauth proxy-logging copy slo versioned_writes proxy-server
+pipeline = catch_errors cache s3api tempauth proxy-logging copy slo versioned_writes symlink proxy-server
```

Comment:
Interesting

from https://github.com/openstack/swift/blob/master/swift/common/middleware/versioned_writes/object_versioning.py
When data is ``PUT`` into a versioned container (a container with the
versioning flag enabled), the actual object is written to a hidden container
and a symlink object is written to the parent container. Every object is
assigned a version id. This id can be retrieved from the
``X-Object-Version-Id`` header in the PUT response.